src/main/java/org/openstreetmap/atlas/geography/atlas/command/AtlasFindByIdSubCommand.java

```diff
@@ -0,0 +1,126 @@
+package org.openstreetmap.atlas.geography.atlas.command;
+
+import java.io.PrintStream;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasEntity;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+
+/**
+ * Searches a collection of atlases for a set of ids. The atlases containing each id are reported.
+ * Optionally, all atlases that contain one of the ids are joined and output to a single atlas file.
+ *
+ * @author bbreithaupt
+ */
+public class AtlasFindByIdSubCommand extends AbstractAtlasSubCommand
+{
+    private static final Command.Switch<Set<String>> FEATURE_ID_PARAMETER = new Command.Switch<>(
+            "id", "list of comma-delimited Atlas extended feature identifier",
+            possibleMultipleOSMIdentifier -> Stream.of(possibleMultipleOSMIdentifier.split(","))
+                    .collect(Collectors.toSet()),
+            Command.Optionality.REQUIRED);
+
+    private static final Command.Switch<String> JOINED_OUTPUT_PARAMETER = new Command.Switch<>(
```

Comment:
I wanted to make it distinct from the `OUTPUT_PARAMETER` name that is used in a lot of the other sub commands because it serves a slightly different purpose. 

I will add to the description to say what happens if it is not passed.

Reply:
Cool 👍